bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/dyee012j_5f_NLO_FXFX_M105to160/dyee012j_5f_NLO_FXFX_M105to160_run_card.dat

```diff
@@ -0,0 +1,144 @@
+91.188 = QES_ref_fixed ! fixed Ellis-Sexton reference scale
+1 = muR_over_ref ! ratio of current muR over reference muR
+1 = muF1_over_ref ! ratio of current muF1 over reference muF1
+1 = muF2_over_ref ! ratio of current muF2 over reference muF2
+1 = QES_over_ref ! ratio of current QES over reference QES
+#***********************************************************************
+# Reweight flags to get scale dependence and PDF uncertainty *
+# For scale dependence: factor rw_scale_up/down around central scale *
+# For PDF uncertainty: use LHAPDF with supported set *
+#***********************************************************************
+.true. = reweight_scale ! reweight to get scale dependence
+0.5 = rw_Rscale_down ! lower bound for ren scale variations
+2.0 = rw_Rscale_up ! upper bound for ren scale variations
+0.5 = rw_Fscale_down ! lower bound for fact scale variations
+2.0 = rw_Fscale_up ! upper bound for fact scale variations
+$DEFAULT_PDF_MEMBERS = reweight_PDF ! reweight to get PDF uncertainty
+#***********************************************************************
+# Merging - WARNING! Applies merging only at the hard-event level. *
+# After showering an MLM-type merging should be applied as well. *
+# See http://amcatnlo.cern.ch/FxFx_merging.htm for more details. *
+#***********************************************************************
+3 = ickkw ! 0 no merging, 3 FxFx merging
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) *
+#***********************************************************************
+15 = bwcutoff
+#***********************************************************************
+# Cuts on the jets *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be *
+# considerably softer than the analysis cuts. *
+# (more specific cuts can be specified in SubProcesses/cuts.f) *
+#***********************************************************************
+1 = jetalgo ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+1.0 = jetradius ! The radius parameter for the jet algorithm
+10 = ptj ! Min jet transverse momentum
+-1 = etaj ! Max jet abs(pseudo-rap) (a value .lt.0 means no cut)
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-) *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f) *
+#***********************************************************************
+0 = ptl ! Min lepton transverse momentum
+-1 = etal ! Max lepton abs(pseudo-rap) (a value .lt.0 means no cut)
+0 = drll ! Min distance between opposite sign lepton pairs
+0 = drll_sf ! Min distance between opp. sign same-flavor lepton pairs
+0 = mll ! Min inv. mass of all opposite sign lepton pairs
+105 = mll_sf ! Min inv. mass of all opp. sign same-flavor lepton pairs
```

Comment:
You use this as a lower cut and then run filers during production?

Reply:
I'd rather instead recommend adding the cuts.f for maxmll, since some LHE events would be a waste with current setups.

Reply:
for the max mLL cut, is it possible to just add this in the run card rather than creating a dedicated cut card?

Reply:
No it won't work you need to add the cuts.f
And just out of curiosity, what would be your analysis level mll selections for the signals?

Reply:
OK sure. Could you point us to an example for how to do this please?
The analysis level selection is currently 110 < mll < 150 GeV, however this is still subject to change.

Reply:
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYJetsToLL_jetbinned_vptbinned_NLO/dyellell1j_ZpT-120to280_5f_NLO_FXFX/dyellell1j_ZpT-120to280_5f_NLO_FXFX_cuts.f#L382-L401

This might be helpful? Then I would suggest you to loosen the cut a little bit more 100-160